proposals/type-classes.md

```diff
@@ -171,10 +171,10 @@ transform(listOf(1), { it + 1 }) // does not compile: No `Functor<List>` instanc
 
 ## Language changes
 
-- Add `with` to require instances evidences in both function and class/object declarations
-- Add `extension` to provide instance evidences for a given type class
+* Add `with` to require evidence of type class instances in both function and class/object declarations.
```

Comment:
* Standardising on "*" for unordered list style (MD004/ul-style).
* Juxtaposed plural nouns are [hard to read](https://english.stackexchange.com/questions/258111/two-plural-nouns-in-a-row) - reworded to avoid this.